### PR TITLE
Various fixes for pinephone, ofono and fingerterm

### PR DESCRIPTION
**Pull request by Tofee**
This fixes some bad UI overlap for some compat apps

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>

### CODE REVIEW

**Comment by Tofee** [comment]
Please double-check the oFono recipe change, I hope I didn't break anything here

File: meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend [hunk 1]
```diff
@@ -1,37 +1,56 @@
...
 
-S = "${WORKDIR}/git/ofono"
+S = "${WORKDIR}/git"
+S_halium = "${WORKDIR}/git/ofono"
 
 # Can't build out of tree right now so we have to build in tree
 B = "${S}"
 
-EXTRA_OECONF_append = " --disable-sailfish-pushforwarder --enable-sailfish-rilmodem"
+EXTRA_OECONF_halium_append = " --disable-sailfish-pushforwarder --enable-sailfish-rilmodem"
```
Comment (shr-project) [inline meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend]: Is this really what you wanted to do?
EXTRA_OECONF_append_halium
makes much more sense and is equivalent of what was there before.

File: meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend [hunk 2]
```diff
@@ -1,37 +1,56 @@
...
 
 # this version does't support it:
 # ERROR: ofono-1.19+gitAUTOINC+b5ed6d16db-r0 do_configure: QA Issue: ofono: configure was passed unrecognised options: --enable-external-ell [unknown-configure-option]
 # enabled in oe-core with 1.29 version
-EXTRA_OECONF_remove = "--enable-external-ell"
+EXTRA_OECONF_halium_remove = "--enable-external-ell"
```
Comment (shr-project) [inline meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend]: The same issue as the append above.

File: meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend [hunk 3]
```diff
@@ -1,37 +1,56 @@
...
 
 do_install_append() {
```
Comment (shr-project) [inline meta-luneos/recipes-connectivity/ofono/ofono_%.bbappend]: SERVICE_FILE = "ofono.service"
SERVICE_FILE_halium = "ofono-halium.service"

and
cp -v ${WORKDIR}/${SERVICE_FILE} ${D}${systemd_unitdir}/system/ofono.service
to make sure that it's always copied or the do_install fails (it should always exist, right?)

**Pull request merged**